### PR TITLE
Adds service related MCQ's

### PR DESCRIPTION
**Pull request by rpdasilva**
connects #210

### CODE REVIEW

File: philip-services-mcq.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+1.
+    ```typescript
+    class MyComponent {
+      private http: Http;
+    }
+    ```
+1.
+    ```typescript
+    class MyComponent {
+      constructor(http: Http) {
```
Comment (winkerVSbecks) [inline philip-services-mcq.md]: should this be `constructor(private http: Http) {`?

Comment (winkerVSbecks) [inline philip-services-mcq.md]: never mind. ISWYDT

File: philip-services-mcq.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+<!--
+	Correct answer is 3.
+
+  1, 2 - Services can only be injected once they've been explicitly registered inside of a `providers` array. Angular's DI does not implicitly construct classes on request.
+-->
+Injecting a service into a class accomplishes which of the following outcomes?
+
+1. A new instance of the service is constructed and made available to the class. The newly constructed instance of the service is then made available to the class.
```
Comment (winkerVSbecks) [inline philip-services-mcq.md]: have you considered rewording this to `A new instance of the service is constructed which is then made available to the class.`

File: philip-services-mcq.md (hunk 1 above)
Comment (gvwilson) [inline philip-services-mcq.md]: Will learners have the same question that you did? Or can this be a question in its own right?  If it took you two looks...

File: philip-services-mcq.md (hunk 2 above)
Comment (rpdasilva) [inline philip-services-mcq.md]: That does sound better

File: philip-services-mcq.md (hunk 1 above)
Comment (rpdasilva) [inline philip-services-mcq.md]: Would be curious to hear as well - It was intentional to leave out `private` as it's not explicitly required, but if it took a second look, perhaps it's best to stay as simple as possible.

Comment (rpdasilva) [inline philip-services-mcq.md]: @gvwilson The following question actually addresses this exact difference

**Comment by jamesbs** [comment]
Merged into branch reorg-2017 (e7f706d128fec79e69a88a37bd0204781747709d)

**Pull request closed without merging**